src/inmanta/server/services/environmentservice.py

```diff
@@ -510,7 +510,6 @@ async def environment_delete(self, environment_id: uuid.UUID) -> None:
                     env.delete_cascade(connection=connection),
                 )
 
-            self.resource_service.close_resource_action_logger(environment_id)
```

Comment:
What are the consequences of not doing this anymore? I think just a tiny (negligible) memory leak?

Reply:
True. The file handler will remain open although the environment doesn't exist anymore. It will be closed on the next restart of the server.